### PR TITLE
Add explict integration list to migration guide

### PR DESCRIPTION
**Pull request by rkoron007**

### CODE REVIEW

File: docs/source/migration.mdx [hunk 1]
```diff
@@ -280,7 +282,18 @@ The Apollo Server core team no longer maintains the following integration packag
 * [`apollo-server-cloudflare`](https://www.npmjs.com/package/apollo-server-cloudflare)
 * [`apollo-server-azure-functions`](https://www.npmjs.com/package/apollo-server-azure-functions)
 
-If you use one of these packages, see [the community integration index](./integrations/integration-index/) to see if an integration exists for your environment yet, or [build your own](./integrations/building-integrations).
+Instead, we now work with the broader community to maintain the following open-source integrations for Apollo Server:
+
+| Web Framework | Integration Package |
```
Comment (glasser) [inline docs/source/migration.mdx:287]: Could you move this into a component shared with integration-index? I hadn't thought of just putting the full table here which does look nice, but would also be nice to not have to update two pages.

File: docs/source/migration.mdx [hunk 2]
```diff
@@ -280,7 +282,18 @@ The Apollo Server core team no longer maintains the following integration packag
...
+|--------|-------------|
+| [AWS Lambda](https://aws.amazon.com/lambda/) | [`@as-integrations/aws-lambda`](https://www.npmjs.com/package/@as-integrations/aws-lambda) |
+| [Fastify](https://fastify.io/) | [`@as-integrations/fastify`](https://www.npmjs.com/package/@as-integrations/fastify) |
+| [Hapi](https://hapi.dev/) | [`@as-integrations/hapi`](https://www.npmjs.com/package/@as-integrations/hapi) |
+| [Koa](https://koajs.com/) | [`@as-integrations/koa`](https://www.npmjs.com/package/@as-integrations/koa) |
+| [Next.js](https://nextjs.org) | [`@as-integrations/next`](https://www.npmjs.com/package/@as-integrations/next) |
+| [Nuxt](https://v3.nuxtjs.org/) / [h3](https://github.com/unjs/h3) | [`@as-integrations/h3`](https://www.npmjs.com/package/@as-integrations/h3) |
+
+> See [the community integration index](./integrations/integration-index/) for the most up-to-date list of current integrations. If an integration doesn't exist for your framework, you can always [build your own](./integrations/building-integrations)!
```
Comment (glasser) [inline docs/source/migration.mdx:296]: and if you do that you probably don't even need the link to integration-index here.

**Review by glasser — CHANGES_REQUESTED** [review]

File: docs/source/migration.mdx (hunk 1 above)
Comment (rkoron007) [reply docs/source/migration.mdx]: Oh, brilliant idea! 🧠 Done!

File: docs/source/migration.mdx (hunk 2 above)
Comment (rkoron007) [reply docs/source/migration.mdx]: Removed this link too :)

File: docs/source/integrations/integration-index.mdx [hunk 3]
```diff
@@ -12,15 +14,9 @@ The larger community maintains the following open-source integrations for Apollo
...
-<!-- prettier-ignore -->
-| Web Framework | Integration Package |
-|--------|-------------|
-| [AWS Lambda](https://aws.amazon.com/lambda/) | [`@as-integrations/aws-lambda`](https://www.npmjs.com/package/@as-integrations/aws-lambda) |
-| [Fastify](https://fastify.io/) | [`@as-integrations/fastify`](https://www.npmjs.com/package/@as-integrations/fastify) |
-| [Hapi](https://hapi.dev/) | [`@as-integrations/hapi`](https://www.npmjs.com/package/@as-integrations/hapi) |
-| [Koa](https://koajs.com/) | [`@as-integrations/koa`](https://www.npmjs.com/package/@as-integrations/koa) |
-| [Next.js](https://nextjs.org) | [`@as-integrations/next`](https://www.npmjs.com/package/@as-integrations/next) |
-| [Nuxt](https://v3.nuxtjs.org/) / [h3](https://github.com/unjs/h3) | [`@as-integrations/h3`](https://www.npmjs.com/package/@as-integrations/h3) |
+The larger community maintains the following open-source integrations for Apollo Server:
```
Comment (glasser) [inline docs/source/integrations/integration-index.mdx:17]: Looks like you have this text twice

**Review by glasser — APPROVED** [review]
remove that duplicated line but otherwise looks great!

File: docs/source/integrations/integration-index.mdx (hunk 3 above)
Comment (rkoron007) [reply docs/source/integrations/integration-index.mdx:15]: Thank you for catching!

**Pull request merged**